### PR TITLE
Add Support for Opensearch Dashboards

### PR DESCRIPTION
**Pull request by raihankhan**
Signed-off-by: raihankhan <raihan@appscode.com>

### CODE REVIEW

File: apis/dashboard/v1alpha1/types.go [hunk 1]
```diff
@@ -88,9 +88,10 @@ const (
...
 const (
-	DashboardVolumeCustomConfig            DashboardVolumeName = "dashboard-custom-config"
-	DashboardVolumeOperatorGeneratedConfig DashboardVolumeName = "dashboard-temp-config"
-	DashboardVolumeConfig                  DashboardVolumeName = "dashboard-config"
-	DashboardVolumeKibanaServer            DashboardVolumeName = "kibana-server"
-	DashboardVolumeDatabaseClient          DashboardVolumeName = "database-client"
+	DashboardVolumeCustomConfig               DashboardVolumeName = "dashboard-custom-config"
+	DashboardVolumeOperatorGeneratedConfig    DashboardVolumeName = "dashboard-temp-config"
+	DashboardVolumeConfig                     DashboardVolumeName = "dashboard-config"
+	DashboardVolumeKibanaServer               DashboardVolumeName = "kibana-server"
```
Comment (kamolhasan) [inline apis/dashboard/v1alpha1/types.go:94]: "server-tls"

File: apis/dashboard/v1alpha1/elasticsearchdashboard_types.go [hunk 2]
```diff
@@ -25,13 +25,6 @@ import (
 	ofst "kmodules.xyz/offshoot-api/api/v1"
 )
 
-const (
```
Comment (kamolhasan) [inline apis/dashboard/v1alpha1/elasticsearchdashboard_types.go:28]: Keep them here.

**Review by kamolhasan — CHANGES_REQUESTED** [review]

File: apis/dashboard/v1alpha1/elasticsearchdashboard_types.go (hunk 2 above)
Comment (raihankhan) [reply apis/dashboard/v1alpha1/elasticsearchdashboard_types.go]: re-added the constants here

File: apis/dashboard/v1alpha1/types.go (hunk 1 above)
Comment (raihankhan) [reply apis/dashboard/v1alpha1/types.go]: renamed

**Review by kamolhasan — DISMISSED** [review]
LGTM!

**Review by kamolhasan — APPROVED** [review]

**Review by kamolhasan — APPROVED** [review]

**Pull request merged**